controllers/UserManagement/ManageOrgsController.js

```diff
@@ -0,0 +1,35 @@
+const BaseController = require('../BaseController');
+const { orgLookup } = require('../../services/UserManagement/OrganisationSearchService');
+
+class ManageOrgsController extends BaseController {
```

Comment:
I think this should be called "OrganisationsController". I appreciate it's close to "OrganisationController" but the singular vs plural tells us something. Perhaps we could add a comment indicating what this page is doing:

```suggestion
// Page that lists multiple organisation or redirects if there is only one
class OrganisationsController extends BaseController {
```